examples/webgpu_postprocessing_afterimage.html

```diff
@@ -21,62 +27,145 @@
 		<script type="module">
 
 			import * as THREE from 'three';
-			import { pass } from 'three/tsl';
+			import { instancedBufferAttribute, mod, pass, texture, float, time, vec2, vec3, vec4, sin, cos } from 'three/tsl';
 			import { afterImage } from 'three/addons/tsl/display/AfterImageNode.js';
 
 			import { GUI } from 'three/addons/libs/lil-gui.module.min.js';
+			import Stats from 'three/addons/libs/stats.module.js';
 
-			let camera, scene, renderer;
-			let mesh, postProcessing, combinedPass;
+			let camera, scene, renderer, particles, stats;
+			let postProcessing, afterImagePass, scenePass;
 
 			const params = {
 
-				damp: 0.96
+				damp: 0.8,
+				enabled: true
 
 			};
 
 			init();
-			createGUI();
 
 			function init() {
 
-				renderer = new THREE.WebGPURenderer( { antialias: true } );
+				renderer = new THREE.WebGPURenderer();
 				renderer.setPixelRatio( window.devicePixelRatio );
 				renderer.setSize( window.innerWidth, window.innerHeight );
 				renderer.setAnimationLoop( animate );
 				document.body.appendChild( renderer.domElement );
 
-				camera = new THREE.PerspectiveCamera( 70, window.innerWidth / window.innerHeight, 1, 1000 );
-				camera.position.z = 400;
+				camera = new THREE.PerspectiveCamera( 60, window.innerWidth / window.innerHeight, 1, 10000 );
+				camera.position.z = 1000;
 
 				scene = new THREE.Scene();
-				scene.fog = new THREE.Fog( 0x000000, 1, 1000 );
 
-				const geometry = new THREE.TorusKnotGeometry( 100, 30, 100, 16 );
-				const material = new THREE.MeshNormalMaterial();
-				mesh = new THREE.Mesh( geometry, material );
-				scene.add( mesh );
+				const sprite = new THREE.TextureLoader().load( 'textures/sprites/circle.png' );
+
+				// geometry
+
+				const radius = 600;
+				const count = 50000;
+
+				const vertex = new THREE.Vector3();
+				const color = new THREE.Color();
+
+				const colors = [];
+				const vertices = [];
+				const timeOffsets = [];
+
+				for ( var i = 0; i < count; i ++ ) {
+
+					getRandomPointOnSphere( radius, vertex );
+					vertices.push( vertex.x, vertex.y, vertex.z );
+
+					color.setHSL( i / count, 0.7, 0.7, THREE.SRGBColorSpace );
+					colors.push( color.r, color.g, color.b );
+
+					timeOffsets.push( i / count );
+			
+				}
+
+				const positionAttribute = new THREE.InstancedBufferAttribute( new Float32Array( vertices ), 3 );
+				const colorAttribute = new THREE.InstancedBufferAttribute( new Float32Array( colors ), 3 );
+				const timeAttribute = new THREE.InstancedBufferAttribute( new Float32Array( timeOffsets ), 1 );
+
+				// material and TSL
+
+				const material = new THREE.SpriteNodeMaterial( { blending: THREE.AdditiveBlending, depthWrite: false, transparent: true } );
```

Comment:
I had to set `transparent` to `true` since the default value of `SpriteNodeMaterial` is `false`. 

However, `SpriteMaterial.transparent` is `true` by default.

It seems the copy of default values in `NodeMaterial.setDefaultValues()` does not work as expected.

https://github.com/mrdoob/three.js/blob/88062edb091598ce32696f3c0a11d3a73c840295/src/materials/nodes/NodeMaterial.js#L1028-L1060

Reply:
> If not, it seems the copy of default values in NodeMaterial.setDefaultValues() does not work in all cases.

Yes, sprites should be transparent by default.

It would just define values ​​that do not exist in the class in order to save some computation when cloning the classes, of course this could be solved by testing, but I think this approach is a hack and should be discontinued at some point and replaced by something like https://github.com/mrdoob/three.js/pull/28328